src/components/card/card.tsx

```
@@ -0,0 +1,88 @@
+    if (isHeader && isAcceptKey) {
+      ev.preventDefault();
+      toggleBody();
+    }
+  };
+
+  return (
+    <div className={classNames('card', className)} id={id}>
+      {toggleBody ? (
+        <div className="card-headerContainer">
+          <div
+            role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)